js/renderBot.js

```diff
@@ -166,12 +166,43 @@ function renderBot(directLineKey, botSecret) {
         }
     });
 
+    // calling session storage data to other pages on Bot 
+    $(function storeddata() {
+        if (sessionStorage["myKey"] != null) {
+            var contentsOfOldDiv = JSON.parse(sessionStorage["myKey"]);
+            $(".wc-message-group-content").html(contentsOfOldDiv);
+        }
+    });
+
+    $(function appendmessage() {
+        var a;
+        newMessages = document.getElementsByClassName("wc-message-groups")[0];
+        newMessages.addEventListener('DOMNodeInserted', function (evt) {
+            if (newMessages.getElementsByClassName("wc-message-group-content") !== undefined) {
+                var msgList = document.getElementsByClassName('wc-message-wrapper list').length;
+                if (isFirstMessage === 0 && $(evt.target).attr('class') === 'wc-message-wrapper list') {
+                    isFirstMessage = 1;
+                    var targetNode = document.getElementsByClassName('wc-message-wrapper list')[0];
+                    newMessages.getElementsByClassName("wc-message-group-content")[0].removeChild(targetNode);
+                    newMessages.getElementsByClassName("wc-message-group-content")[0].appendChild(targetNode);
```

Comment:
Use jquery chaining

Reply:
Chaining won't work in the case of remove and append.
![chaning error](https://user-images.githubusercontent.com/44663443/55329595-a2908880-54ac-11e9-9562-869ac796ac33.png)